WORKSPACE

```diff
@@ -34,20 +34,20 @@ bind(
 
 # When updating envoy sha manually please update the sha in istio.deps file also
 #
-# Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
-# envoy-wasm commit date: 08/13/2019
+# Determine SHA256 `wget https://github.com/istio/envoy/archive/COMMIT.tar.gz && sha256sum COMMIT.tar.gz`
+# envoy commit date: 08/13/2019
 # bazel version: 0.28.1
 ENVOY_SHA = "3f90336c39250c472cfd51b157c305582b7a6725"
```

Comment:
I think this needs to change too.

Reply:
Err, or the other shas were wrong and this PR fixed them

Reply:
No it doesn't. I already pulled that commit when I branched `release-1.3` on the freeze date.

Reply:
The archive SHA256 needs to be updated only because the file paths are different `envoy-*` instead of `envoy-wasm-*`, but the code is identical.